src/prefs/MusicInformationRetrievalPrefs.cpp

```diff
@@ -0,0 +1,9 @@
+#include "MusicInformationRetrievalPrefs.h"
+
+ChoiceSetting UseMirResultToConfigureProject {
+   wxT("/GUI/UseMirResultToConfigureProject"),
+   { ByColumns,
+     { {}, {}, {} },
```

Comment:
This associated empty user-visible strings with the identifiers.  That's wrong.  But what happened in later commits? ...